terraform/testnet/variables.tf

```diff
@@ -38,6 +38,11 @@ variable "ssh_pub_key" {
   description = "SSH public key to configure for bastion and vault access"
 }
 
+variable "validator_lite_mode" {
```

Comment:
should this also toggle different scaling factors for the nodepool sizes?

Reply:
yeah I thought about this, it would look ugly so I didn't do it but I guess it's fair...will update